app/components/doi-subject.js

```diff
@@ -0,0 +1,139 @@
+import Component from '@ember/component';
+import ISO6391 from 'iso-639-1';
+import URI from 'urijs';
+import { isBlank } from '@ember/utils';
+// import OECD from 'helpers/oecd-list';
```

Comment:
Was there an idea to have this in another place? as I sort of like that idea rather than hardcoding in subject component itself.
At any rate if not, remove the dead code comments

Reply:
It would be nice to create a npm package that had the list. but no at the moment we decide to put all the subjects in the component.